data/python.gram

```diff
@@ -317,27 +362,39 @@ class Parser(Parser):
                 self._tokenizer.get_lines(list(range(start[0], end[0] + 1)))
             )
 
-        return  SyntaxError(
-            message,
-            (self.filename, start[0], start[1], line)
-        )
+        args = (self.filename, start[0], start[1] + 1, line)
```

Comment:
Can we add a comment on why now we have an offset of 1?

Reply:
Will do. This is related to @edemaine comment that tokenize index at 0 and SyntaxError at 1.

Reply:
Added a comment